src/systems/hubs-systems.ts

```diff
@@ -245,7 +246,8 @@ export function mainTick(xrFrame: XRFrame, renderer: WebGLRenderer, scene: Scene
   hubsSystems.gainSystem.tick();
   hubsSystems.nameTagSystem.tick();
   simpleWaterSystem(world);
-
+  textSystem(world);
```

Comment:
Its very crude but we may want to add a blank line above this system to make it clear its a bit "special". Or maybe better to just add a comment like `// all systems that update text properties should run before this`